deploy/cr.yaml

```diff
@@ -66,3 +66,7 @@ spec:
   # Provide the name of the configmap which holds your .pem in a field called 'certs'
   # If this is not set the default embedded certificates on the images will be used
   #trustedCAs: name-of-my-ca-configmap
+  # Sets a NetworkZone for the OneAgent (optional)
+  # If this is not set the default NetworkZone is 'default'
```

Comment:
I'd say that the default is a technical detail and we shouldn't mention it here. It doesn't matter to us either, since we just don't set the installer argument in this case.